Integrations/integration-ThreatX.yml

```diff
@@ -0,0 +1,711 @@
+                actor['interval_time_stop'] = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(actor['interval_time_stop']))
+                actor['fingerprint']['last_seen'] = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(actor['fingerprint']['last_seen']))
+                # Set Dbot scores
+                if e_risk >= 70:
+                    dbot_scores.append({
+                            'Vendor' : 'ThreatX',
+                            'Indicator' : ipdot,
+                            'Type' : 'ip',
+                            'Score' : 3,
+                            'isTypedIndicator': True,
+                            'Malicious' : {
+                                'Vendor' : 'ThreatX',
+                                'Description' : 'ThreatX risk score is ' + str(e_risk),
+                                'IP' : ipdot
+                            }
+                        })
+                elif e_risk > 10:
+                    dbot_scores.append({
+                            'Vendor' : 'ThreatX',
+                            'Indicator' : ipdot,
+                            'Type' : 'ip',
+                            'Score' : 2,
+                            'isTypedIndicator': True
+                        })
+                else:
+                    dbot_scores.append({
+                            'Vendor' : 'ThreatX',
+                            'Indicator' : ipdot,
+                            'Type' : 'ip',
+                            'Score' : 0,
+                            'isTypedIndicator': True
+                        })
+
+            ec = {
+                    'Threatx.Entity(val.ID && val.ID === obj.ID)': {
+                        'ID': entity['id'],
+                        'Name': entity['codename'],
+                        'IP': iplist,
+                        'Risk': e_risk
+                    },
+                    'DBotScore': dbot_scores
+                }
+
+            output.append({
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': entity,
+                    'EntryContext': ec})
+
+        demisto.results(output)
+
+    @logger
+    def get_entity_notes(entity_id):
+        commands = {
+                'command': 'notes',
+                'id': entity_id
+            }
+
+        return http_request('/entities', commands)
+
+    @logger
+    def get_entity_notes_command():
+        entity_id = demisto.args().get('entity_id', None)
+        results = get_entity_notes(entity_id)
+
+        # Reverse sort the list by timestamp
+        sorted_results = sorted(results, key=itemgetter('timestamp'), reverse=True)
+
+        # Replace dates with pretty format
+        for note in sorted_results:
+            note['timestamp'] = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(note['timestamp']))
```

Comment:
Need to verify `timestamp` is in the dictionary.

Reply:
done